Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-grid-row/SubmissionGridRow.module.scss

```diff
@@ -3,101 +3,113 @@
 @use 'src/styles/variables';
 @use 'src/styles/responsive';
 @use 'src/styles/shadows';
+@use 'src/styles/spacings';
+@use 'src/styles/font-weights.scss';
+@use 'src/styles/border-radiuses.scss';
 
-.textAlignRight {
-  text-align: right;
-}
-
-.lightSubmissionsGridHeader {
-  background: colors.$wrappers-border-color;
-  box-shadow: shadows.$dp-shadow-1;
-}
-
-.darkSubmissionsGridHeader {
-  background: #1c1c17;
-  box-shadow: shadows.$dp-shadow-4;
-}
-
-.submissionsGridHeader {
+.row {
   @extend %font-family-roboto;
 
-  align-content: start;
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
-  padding: 8px 25px;
+  height: 100%;
+}
 
-  div {
-    padding: 0 10px;
-  }
+.lightRow {
+  background: colors.$horizontal-line-gray;
+  border-bottom: 1px solid colors.$wrappers-border-color;
 }
 
-.smallColumn {
-  flex: 1;
-  width: 50px;
+.darkRow {
+  background: colors.$primary-dark-grey-color;
+  border-bottom: 1px solid #212328;
 }
 
-.mediumColumn {
-  flex: 4;
-  width: 180px;
+.redirectButton {
+  background-color: inherit;
+  border: 0;
+  color: colors.$light-blue-color;
+  cursor: pointer;
+  font: inherit;
+  font-size: variables.$f-size-16;
+  margin-top: spacings.$sp-4;
+  padding-left: 0;
+  text-align: left;
 }
 
-.wideColumn {
-  flex: 6;
-  flex-direction: column;
-  width: 200px;
+.link {
+  @extend %r-heading-primary;
 
-  .redirectButton {
-    padding: 0;
-    text-align: left;
-  }
+  color: colors.$light-blue-color;
+  display: inline-block;
+  margin: spacings.$sp-8;
 }
 
-.smallColumn,
-.mediumColumn,
-.wideColumn {
+.strategy {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  gap: spacings.$sp-12;
+  height: 100%;
   justify-content: start;
-  text-align: left;
 }
 
-.lightRow {
-  background: colors.$horizontal-line-gray;
-  border-bottom: 1px solid colors.$wrappers-border-color;
+.competeIcon {
+  margin: 0 spacings.$sp-12;
 }
 
-.darkRow {
-  background: colors.$background-color-footer;
-  border-bottom: 1px solid #212328;
+.competeIconWrapper {
+  position: relative;
 }
 
-.container {
-  @extend %font-family-roboto;
+.executionResultContainer {
+  display: flex;
+  flex-direction: row;
+  gap: spacings.$sp-18;
+}
 
+.maxTimeUsed,
+.maxMemoryUsed {
   align-items: center;
   display: flex;
-  flex-wrap: wrap;
-  justify-content: flex-start;
-  padding: 15px;
+  flex-direction: column;
 
-  .smallColumn,
-  .wideColumn {
-    padding: 0 10px;
+  .memoryIcon,
+  .timeIcon {
+    color: colors.$dark-base-color-100;
```

Comment:
isnt this color from the theme dynamic colors? if so this would not change when switching the theme and it should come as classname style from useTheme hook

Reply:
![image](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/19411274/6a81afd1-7cb0-48d0-8ce8-9655e2db3fc6)
![image](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/19411274/72d67205-51ce-4a9f-a2c2-f0dd407e1885)

Looks okay on both light and dark themes